### DISCUSSION TITLE
6.0 Release?

### DISCUSSION THREAD

**Discussion by PlanetOrca** [Xeloboyo/Factoriodustry#26] [General]
Hi Xelo, 

My friends and I _really_ enjoy this texture pack. I stealthily installed this on one of their computers and they thought it had autoupdated and that it looked way better. If you wanted motivation to continue working on this, consider my friend group your motivation. Maybe, just maybe, there could be a little cash in it for you as well.

Thanks Xeloboyo,
PlanetOrca
Wildfire
gunzer
ajmar

**Comment by sonht-0555** [comment]
contact me, I have full mod v6.
Hoangtuanson91@gmail.com

**Reply by sonht-0555** [reply]
![image](https://user-images.githubusercontent.com/26568422/124392481-10792980-dd20-11eb-8d12-1e9da71798c5.png)

**Reply by PlanetOrca** [reply]
wtf bro I emailed you like over a month ago and you didn't respond
